.github/workflows/windows_2022.yml

```diff
@@ -0,0 +1,30 @@
+name: Windows (2022)
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - develop
+
+concurrency:
+  group: windows-${{ github.event_name }}-${{ github.ref_name }}-2022
+  cancel-in-progress: true
+
+permissions: read-all
+
+jobs:
+  Ubuntu2404:
```

Comment:
Is this correct, or should it be Windows2022?

Reply:
Good catch, thanks!
Since it doesn't have negative effect on the UI (because `name` is provided), I will fix it in the follow-up PR